<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAccounts</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Accounts_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_Username"/>
    <addaction name="actionCopy_Password"/>
    <addaction name="actionCopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionOpen_Accounts_File">
   <property name="text">
    <string>Open Accounts File</string>
   </property>
  </action>
  <action name="actionCopy_Username">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy Username</string>
   </property>
  </action>
  <action name="actionCopy_Password">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy Password</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
